package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "stats-lite",
-  "version": "2.1.0",
+  "version": "2.1.1",
```

Comment:
We should probably bump minor instead of patch since this adds new functionality

Reply:
Bump to 2.2.0 and rebased